GSA/digital-gov-drupal
# Digital-31: sitemap path auto

Ticket: [Digital-31](https://cm-jira.usa.gov/browse/DIGITAL-31)

Description:
This Pr is setting up 2 things... URL naming conventions and also setting up the sitemap.xml module 

Testing: 
1. create a new page for each content type, compare the URL output to the current live site to ensure that paths are correct
2. visit `.sitemap.xml` and check that sitemap is loading and that the content that was created are showing up as well.
